<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>widget test</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/icons/icons/iconfont-wo.png</normaloff>:/icons/icons/iconfont-wo.png</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>TAB</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>-1</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>MDI</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QMdiArea" name="mdiArea"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>890</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTAB">
    <property name="title">
     <string>TAB功能</string>
    </property>
    <addaction name="action_widget_insert"/>
    <addaction name="action_widget_indep"/>
    <addaction name="separator"/>
    <addaction name="action_mainwindow_insert"/>
    <addaction name="action_mainwindow_indep"/>
   </widget>
   <widget class="QMenu" name="menuMDI">
    <property name="title">
     <string>MDI功能</string>
    </property>
    <addaction name="action_new_widget"/>
    <addaction name="action_open_widget"/>
    <addaction name="separator"/>
    <addaction name="action_cut"/>
    <addaction name="action_copy"/>
    <addaction name="action_paste"/>
    <addaction name="action_font"/>
    <addaction name="separator"/>
    <addaction name="action_save"/>
    <addaction name="action_close_all"/>
   </widget>
   <addaction name="menuTAB"/>
   <addaction name="menuMDI"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionMDI"/>
   <addaction name="action_cascade"/>
   <addaction name="action_tile"/>
   <addaction name="separator"/>
   <addaction name="action_quit"/>
  </widget>
  <action name="action_widget_insert">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icons/icons/chuangkou_1.png</normaloff>:/icons/icons/chuangkou_1.png</iconset>
   </property>
   <property name="text">
    <string>嵌入widget</string>
   </property>
  </action>
  <action name="action_widget_indep">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icons/icons/tubiaozhizuomobanyihuifu-.png</normaloff>:/icons/icons/tubiaozhizuomobanyihuifu-.png</iconset>
   </property>
   <property name="text">
    <string>独立widget</string>
   </property>
  </action>
  <action name="action_mainwindow_insert">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icons/icons/program-code-full.png</normaloff>:/icons/icons/program-code-full.png</iconset>
   </property>
   <property name="text">
    <string>嵌入mainwindow</string>
   </property>
  </action>
  <action name="action_mainwindow_indep">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icons/icons/chuangkou.png</normaloff>:/icons/icons/chuangkou.png</iconset>
   </property>
   <property name="text">
    <string>独立mainwindow</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icons/icons/tuichu.png</normaloff>:/icons/icons/tuichu.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_new_widget">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icons/icons/chuangkou_1.png</normaloff>:/icons/icons/chuangkou_1.png</iconset>
   </property>
   <property name="text">
    <string>新建窗体</string>
   </property>
  </action>
  <action name="action_open_widget">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icons/icons/tubiaozhizuomobanyihuifu-.png</normaloff>:/icons/icons/tubiaozhizuomobanyihuifu-.png</iconset>
   </property>
   <property name="text">
    <string>打开窗体</string>
   </property>
  </action>
  <action name="action_save">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icons/icons/iconfont-wo.png</normaloff>:/icons/icons/iconfont-wo.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="action_close_all">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>关闭全部</string>
   </property>
  </action>
  <action name="action_cut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>剪切</string>
   </property>
  </action>
  <action name="action_copy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="action_paste">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
  </action>
  <action name="action_font">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>字体设置</string>
   </property>
  </action>
  <action name="actionMDI">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icons/icons/1.png</normaloff>:/icons/icons/1.png</iconset>
   </property>
   <property name="text">
    <string>MDI模式</string>
   </property>
  </action>
  <action name="action_cascade">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icons/icons/2.png</normaloff>:/icons/icons/2.png</iconset>
   </property>
   <property name="text">
    <string>级联展开</string>
   </property>
  </action>
  <action name="action_tile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icons/icons/3.png</normaloff>:/icons/icons/3.png</iconset>
   </property>
   <property name="text">
    <string>平铺展开</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
